libtcmu_log.c

```diff
@@ -195,22 +212,20 @@ void tcmu_err_message(const char *funcname, int linenr, const char *fmt, ...)
 	va_start(args, fmt);
 	log_internal(TCMU_LOG_ERROR, funcname, linenr, fmt, args);
 	va_end(args);
-}
-
-void tcmu_warn_message(const char *funcname, int linenr, const char *fmt, ...)
-{
-	va_list args;
-
-	va_start(args, fmt);
-	log_internal(TCMU_LOG_WARN, funcname, linenr, fmt, args);
-	va_end(args);
-}
-
-void tcmu_info_message(const char *funcname, int linenr, const char *fmt, ...)
-{
-	va_list args;
-
-	va_start(args, fmt);
+} 
+
+void tcmu_warn_message(const char *funcname, int linenr, const char *fmt, ...) 
+{ 
+	va_list args; 
+	va_start(args, fmt); 
+	log_internal(TCMU_LOG_WARN, funcname, linenr, fmt, args); 
+	va_end(args); 
+} 
+
+void tcmu_info_message(const char *funcname, int linenr, const char *fmt, ...) 
+{ 
+	va_list args; 
+	va_start(args, fmt); 
 	log_internal(TCMU_LOG_INFO, funcname, linenr, fmt, args);
```

Comment:
From what I can tell the chunk above is just changing the coding style in tcmu_warn_message and tcmu_info_message. If that is correct, could you drop it. Just leave the old style that is used in most of the rest of the code.